### ISSUE TITLE
Could not find rake-12.1.0 in any of the sources

### ISSUE THREAD

**Issue by mmlkrx** [mmlkrx/searchthetimferrissshow#1]
When checking out the repo and starting the container by mounting a volume between host and container, the gems aren't loaded properly and the following error occurs:

```
/app # rspec
Could not find rake-12.1.0 in any of the sources
Run `bundle install` to install missing gems.
```

Not mounting a volume works.

**Comment by mmlkrx** [comment]
Seems like because the `vendor/` directory isn't checked in, as soon as mounting the volume it removes the `vendor/` folder from the container (that has all the gems) and thus all of the gems.

Possible solution would be just installing gems globally and not locally to the project.

**Comment by mmlkrx** [comment]
Fixed in 75b1d2a
